File: 프로그래머스/2/12945. 피보나치 수/피보나치 수.java
class Solution {
    public int solution(int x) {     
        int []  n = new int[x];
        for(int a=0;a<n.length;a++){
            if(a<=2){
                n[a] = a;
                continue;
            }
            n[a] = (n[a-1] + n[a-2])%1234567;
        }
        
        return n[x-1];
    }
}

File: 프로그래머스/2/12945. 피보나치 수/피보나치 수.js
function solution(x) {
    var answer = 0;
    let n = [];
    for(let a=0;a<x;a++){
        if(a<=2){
            n.push(a);
            continue;
        }
    n.push((n[a-1] + n[a-2])%1234567)
    }
        
    return n[x-1];
}
